githubscript/docker.d/worker.yml

```diff
@@ -9,11 +9,6 @@ github_projects:
         github_owner: mozilla-mobile
         github_repo_name: fenix
         contact_github: true
-      android-components:
-        allowed_actions: ["release"]
-        github_token: { "$eval": "GITHUB_TOKEN_WRITE_ACCESS_PROD" }
-        github_owner: mozilla-mobile
-        github_repo_name: android-components
         contact_github: true
```

Comment:
What's happening here? It looks like you should missed one line for deletion? (and multiple instances of the same, below)